numpy/distutils/intelccompiler.py

```diff
@@ -2,7 +2,7 @@
 
 from distutils.unixccompiler import UnixCCompiler
 from numpy.distutils.exec_command import find_executable
-from numpy.distutils.msvc9compiler import MSVCCompiler
+from distutils.msvc9compiler import MSVCCompiler
```

Comment:
However, the patch is from Intel, so I'm inclined to go with what is here and let @cgohlke test it later, he hasn't tested with the intel compiler yet.